repo/pyproject.toml

```diff
@@ -11,8 +11,8 @@ name = "tuf-on-ci"
 description = "TUF-on-CI repository tools, intended to be executed on a CI system"
 readme = "README.md"
 dependencies = [
-  "securesystemslib[awskms, azurekms, gcpkms, sigstore, pynacl] ~= 0.31.0",
-  "tuf ~= 3.1",
+  "securesystemslib[awskms, azurekms, gcpkms, sigstore, pynacl] ~= 1.1.0",
+  "tuf ~= 5.0",
```

Comment:
There are some pretty big bumps here, do we have any insight into if there are breaking changes that would affect us?